bundles/org.palladiosimulator.simulizar.action/src-man/org/palladiosimulator/simulizar/action/interpreter/ActionRuntimeState.java

```diff
@@ -69,7 +73,7 @@ public void setRuntimeStateModel(AbstractSimuLizarRuntimeState passedState) {
         private ControllerCallInputVariableUsageCollection controllerCallVariableUsages = EMPTY_VARIABLE_USAGE_COLLECTION;
         private boolean isAsync = false;
 
-        private TransientEffectInterpreterBuilder(RoleSet roleSet, AdaptationBehaviorRepository repository) {
+        private TransientEffectInterpreterBuilder(RoleSet roleSet, AdaptationBehaviorRepository repository                ) {

```

Comment:
Empty space

Reply:
worked in review finding; reformatted file using Palladio code formatter